DEV_SETUP.md

```diff
@@ -167,28 +171,54 @@ Create the shared directory.  If you have not modified `SHARED_DRIVE_ROOT`, then
 Once you have completed the above steps, you can use Docker to build
 and run all of the service containers. There are detailed instructions
 for setting up Docker in the [docker folder](docker/README.md). But the
-following should cover the needs of most developers:
+following should cover the needs of most developers.
+
 
+1. Install docker packages.
+
+    ```sh
     $ sudo apt install docker.io
     $ pip install docker-compose
     $ sudo adduser $USER docker
+    ```
 
-Log in as yourself again, to activate membership of the "docker" group:
+1. Log in as yourself again to activate membership of the "docker" group and
+   re-activate your virtualenv.
 
+    ```sh
     $ su - $USER
+    $ workon cchq
```

Comment:
Since more of us are using pyenv, we could also cater to those users `pyenv activate cchq`.

Reply:
Yeah, I was conflicted about this since the rest of the doc seems to be (mostly) virtualenvwrapper-centric. I'll push an improvement.

Reply:
I know you didn't change this, but either this or the virtualenv name recommended when [installing commcare-cloud](https://dimagi.github.io/commcare-cloud/setup/manual_installation.html#manual-setup) _must_ be changed. My preference would be to recommend `hq` here, and possibly also change the recommendation for commcare-cloud to `cloud` (although I think that one will be a little harder to change because there is code expects it).

Reply:
> either this or the virtualenv name recommended when [installing commcare-cloud](https://dimagi.github.io/commcare-cloud/setup/manual_installation.html#manual-setup) _must_ be changed.

@millerdev, are you advocating to add the change to this PR, or would a follow-up PR be sufficient?

Reply:
Follow-up PR would be fine since that name is not new in this PR.

Reply:
@millerdev yeah we ran into this exact problem with setup. I was like WHY ARE BOTH `cchq`??? 🤣